README.md

```diff
@@ -1 +1,141 @@
-# hello-world
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
```

Comment:
- Kindly, the only optional section from the readme is `FAQ` so you forget to add the content of `Future Features`, Please add it to have more professional readme 👌

Reply:
- [ ] As requested by the previous reviewers, the `readme` is missing the **future features section** Kindly add it back 😃 
![futures](https://user-images.githubusercontent.com/99656687/234237615-a5ead4ca-e57a-4174-9e39-dbde2656bc8b.png)